content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -309,6 +349,52 @@ To use OpenTelemetry instrumentation libraries with the Datadog .NET SDK:
 1. Set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
 2. Follow the steps to configure each library, if any, to generate OpenTelemetry-compatible instrumentation via `ActivitySource`
 
+The following example demonstrates how to instrument the `Hangfire` OpenTelemetry integrations with the Datadog .NET SDK:
+
+```csharp
+using System;
+using Hangfire;
+using Hangfire.MemoryStorage;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Instrumentation.Hangfire;
```

Comment:
Just wondering if we should include the packages as well here?

```
  <ItemGroup>
    <PackageReference Include="Hangfire" Version="1.8.14" />
    <PackageReference Include="Hangfire.MemoryStorage" Version="1.8.1.1" />
    <PackageReference Include="OpenTelemetry" Version="1.9.0" />
    <PackageReference Include="OpenTelemetry.Api" Version="1.9.0" />
    <PackageReference Include="OpenTelemetry.Instrumentation.Hangfire" Version="1.9.0-beta.1" />
  </ItemGroup>
```

Reply:
+1

Reply:
So I wanted to stay in parity with how we show code snippets elsewhere. We dont include the dlls that are imported as part of it. I think snippet will suffice for now to atleast help the user get started. 

I will create a separate PR explaining the dependencies that are present and how the OTel SDK is not one of them